.github/workflows/e2e-macos.yml

```diff
@@ -39,39 +42,45 @@ jobs:
     - name: Prepare MacOS
       run: brew install screen
 
-    - name: 🕒 Get Date/Time
-      id: date-time
-      shell: bash
-      run: |
-        echo "value=$(rake datetime)" >> $GITHUB_OUTPUT
-
     - name: 💾 GH Cache node db
-      id: cache
+      id: cache-node
       uses: actions/cache@v3
       with:
-        path: test/e2e/state/node_db/${{ env.NETWORK }}
-        key: node-db-${{ runner.os }}-${{ env.NETWORK }}-${{ steps.date-time.outputs.value }}
-        restore-keys: |
-          node-db-${{ runner.os }}-${{ env.NETWORK }}-
-          node-db-Linux-${{ env.NETWORK }}-
+        path: test/e2e/state/node_db/preprod
+        key: node-db-e2e-macos-preprod
 
     - name: 💾 Cache wallet db
       id: cache-wallet
       uses: actions/cache@v3
       with:
-        path: test/e2e/state/wallet_db/${{ env.NETWORK }}
-        key: wallet-db3-${{ runner.os }}-${{ env.NETWORK }}-${{ steps.date-time.outputs.value }}
-        restore-keys: |
-          wallet-db3-${{ runner.os }}-${{ env.NETWORK }}-
+        path: test/e2e/state/wallet_db/preprod
+        key: wallet-db-e2-macos-preprod
+
+    - name: Fetch preprod snapshot
```

Comment:
the same here. fetching preprod snapshot seems like a repetitive task

Reply:
yeah module windows